threema-connector/pom.xml

```diff
@@ -52,6 +52,18 @@
                         </target>
                     </configuration>
                 </execution>
+				<execution>
+                	<id>delete-threema-msgapi-tool-archive</id>
+                	<phase>validate</phase>
+                	<goals>
+                   		<goal>run</goal>
+                	</goals>
+	                <configuration>
+	                    <target>
+	                        <delete file="${project.basedir}/lib/threema-msgapi-tool.zip"/>
+	                    </target>
+	                </configuration>
+            	</execution>
```

Comment:
code is good; but the executions seem a bit overdone in here and therefore harder to maintain.
I'd rather keep all 3 executions in one block ... it's all just about providing the threema.jar which isn't available as normal maven artifact.

Reply:
Combined all steps into 1